packages/manager/modules/pci/src/projects/new/new.routing.js

```diff
@@ -60,10 +60,16 @@ export default /* @ngInject */ ($stateProvider) => {
         getStepByName: /* @ngInject */ steps => stepName => find(steps, {
           name: stepName,
         }),
-        getStateLink: /* @ngInject */ ($state, getCurrentStep) => (action, inherit = true) => {
+        getStateLink: /* @ngInject */ (
+          $state,
+          getCurrentStep,
+          projects,
+        ) => (action, inherit = true) => {
           switch (action) {
             case 'cancel':
-              return $state.href('pci.projects');
+              return $state.href(projects.length
```

Comment:
Why not redirecting to `pci.project` and add a `redirectTo` in the state ?
Like done in `pci.projects.project.instances` for example https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/src/projects/project/instances/instances.routing.js#L5

IMO, it's not the role of an external page to detect if we should redirect to the onboarding or the list. We want to go to the list, and if there is no project the list redirects to the onboarding.